docs/faq/network/blockchain-architecture.md

```diff
@@ -0,0 +1,103 @@
+- **Decentralization and Governanc**e: Client diversity promotes decentralization in the development and governance of the Ethereum network. It prevents any team or entity from having too much influence over the network's growth.
+
+For Ethereum, **client diversity has proven essential** in maintaining the network's robustness during several incidents. One of the most notable incidents were the Shanghai DoS Attacks in 2016, the OpenEthereum Consensus Bug in 2020, and the Prysm Client Incident in 2023. You can read more about them in the [extended client section](https://docs.luksoverse.io/docs/mainnet/complete-node-guide/blockchain-clients/client-setups) by the community.
+
+### How does Shapella enable validator withdrawals?
+
+Shanghai and Capella (Shapella) are the names of an Ethereum hard fork. Shanghai is the fork's name on the execution client side, and Capella is the upgrade name on the consensus layer client side. The included EIPs can be found [here](https://github.com/ethereum/execution-specs/blob/master/network-upgrades/mainnet-upgrades/shanghai.md). The significant new features enable the **withdrawal of the validator's earnings and stake** from the network.
+
+Validators participating in the network need to put down stakes. The withdrawal fork will **allow individuals to stop running their validators** to exit the blockchain with all their stakes or to **access their earned rewards** while keeping the node running.
+
+### What is the London Update?
+
+Each operation in the EVM requires a certain amount of gas, which is paid in the blockchain's native coin. The cost of gas is a **crucial part of the network's incentive structure**, deterring spam and incentivizing miners to confirm transactions.
+
+With the London Update described in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559), the EVM network has a predictable fee system with two parts: a **base fee and a tip**. The base fee is burned and adjusted up or down depending on network congestion. When the network is **busy**, the **base fee increases**, and when the network is less active, the base fee decreases. The tip, also called the priority fee, is given to the miner as an incentive to include the transaction in the block. The London Update automatically went live at the LUKSO mainnet start.
+
+### What is slashing?
+
+In the Proof of Stake consensus context, the slasher functionality **discourages validators from behaving dishonestly or maliciously**. If a validator behaves in a way that could compromise the network's integrity—like trying to manipulate the transaction history or proposing conflicting blocks—they can be slashed. When a validator gets slashed, a portion of its staked LYX or LYXt is removed, e,g. burned. Additionally, they are **ejected from the validator set**, losing their ability to participate in the consensus process and earn further rewards.
```

Comment:
- add space before / after the dashes
- "Additionally, they are **ejected from the validator set**, losing their ability to participate in the consensus process and earn further rewards." -> we should add that there is a condition related to ejection (less than 16 LYX)

Reply:
we actually have a question for that in an other file, let me link them